__tests__/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,10 @@
-const { humanCatDogYears } = require('../src');
+const humanCatDogYears = require('../src/kata4.humanCatDogYears');
 
-// Look Ma, no handlebars!!!
+describe('humanCatDogYears',() => {
+    const expected1 = [10, 56, 64];
+    const expected2 = [20, 96, 114];
+ it('returns an array of human, cat and dog years when passed human years', () => {
```

Comment:
Indent this `it` block and then you can easily see it's part of the surrounding `describe`.